From c21b4451a22e0a451a184ba4cc55f2e3bcc3dfb5 Mon Sep 17 00:00:00 2001
From: Andreas Schneider <asn@cryptomilk.org>
Date: Mon, 22 Jan 2018 07:47:08 +0100
Subject: [PATCH 49/51] storage: Set all sdcards to visible

If they are visibile to apps the media scanner will scan them.

Change-Id: Icceceafc8469b6b9d1c0e72f8425d7e38ae4a10b
(cherry picked from commit 48dc2e83accb27b99c8a713b95988c25111c392b)
---
 .../core/java/com/android/server/StorageManagerService.java | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/services/core/java/com/android/server/StorageManagerService.java b/services/core/java/com/android/server/StorageManagerService.java
index b4da945148c..910e3e345f6 100644
--- a/services/core/java/com/android/server/StorageManagerService.java
+++ b/services/core/java/com/android/server/StorageManagerService.java
@@ -1178,9 +1178,9 @@ class StorageManagerService extends IStorageManager.Stub
                 vol.mountFlags |= VolumeInfo.MOUNT_FLAG_VISIBLE;
             }
 
-            // Adoptable public disks are visible to apps, since they meet
-            // public API requirement of being in a stable location.
-            if (vol.disk.isAdoptable()) {
+            // Set sdcards to visible to apps. If they are visible media is scanned
+            // and they can be used for other stuff.
+            if (vol.disk.isSd()) {
                 vol.mountFlags |= VolumeInfo.MOUNT_FLAG_VISIBLE;
             }
 
-- 
2.17.1

